csrc/core/macro.h

```diff
@@ -3,28 +3,24 @@
 #ifndef MMDEPLOY_SRC_CORE_MARCO_H_
 #define MMDEPLOY_SRC_CORE_MARCO_H_
 
+#ifndef MMDEPLOY_EXPORT
```

Comment:
Because they are different things. `MMDEPLOY_EXPORT` is used to tell compiler the symbol need to be exported. `MMDEPLOY_API_EXPORTS` is defined using compile flags to let `MMDEPLOY_API` substitude by `MMDEPLOY_EXPORT`; sementically, it means that we are building the library instead of using it.